mlrun/runtimes/sparkjob.py

```diff
@@ -233,7 +230,7 @@ def with_igz_spark(self, igz_version):
 
     def get_pods(self, name=None, namespace=None, driver=False):
         k8s = self._get_k8s()
```

Comment:
@Hedingber why not move such methods (get_pods, ..) to the RuntimeHandler

Reply:
Talked offline, will be done in a separate Refactor-PR that will be targeted to assure client code just activate certain API endpoints, and only server code is interacting with the underlying k8s cluster